spec/index.html

```diff
@@ -297,10 +297,11 @@ <h3>Terms defined by this specification</h3>
         the algorithm.</dd>
       <dt><dfn>normalized dataset</dfn></dt>
       <dd>The immutable, abstract <a>RDF dataset</a> and a set of normalized
-        <a>blank node identifiers</a> that are produced as output by the algorithm.
-        A <a>normalized dataset</a> is a restriction on an <a>RDF dataset</a>
-        where all nodes are labeled, and <a>blank nodes</a> are labeled with canonical <a>blank node identifiers</a>
-        consistent with running this algorithm on a base <a>RDF dataset</a>.
+        <a>blank node identifiers</a> that are produced as consistent with this algorithm.
```

Comment:
Is the first sentence in this paragraph still necessary? The second sentence, ie, the one starting with

```html
A <a>normalized dataset</a> is the combination…
```

says the same thing in a more precise format. I would consider removing the first sentence altogether.